content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -221,66 +212,59 @@ <h3 id="coding-webrtc-for-signaling">Coding WebRTC for signaling</h3>
+<p>More recently, the <a href="http://www.html5rocks.com/en/tutorials/eventsource/basics/"><code>EventSource</code> API</a> has been <a href="http://caniuse.com/#feat=eventsource">widely implemented</a>. This enables server-sent events&mdash;data sent from a web server to a browser client through HTTP. <code>EventSource</code> is designed for one-way messaging, but it can be used in combination with XHR to build a service for exchanging signaling messages. A signaling service passes a message from a caller, delivered by XHR request, by pushing it through <code>EventSource</code> to the callee.</p>
 
-<p><a href="http://www.html5rocks.com/en/tutorials/websockets/basics/">WebSocket</a> is a more natural solution, designed for full duplex client–server communication (messages can flow in both directions at the same time). One advantage of a signaling service built with pure WebSocket or Server-Sent Events (EventSource) is that the back-end for these APIs can be implemented on a variety of web frameworks common to most web hosting packages, for languages such as PHP, Python and Ruby.</p>
+<p><a href="http://www.html5rocks.com/en/tutorials/websockets/basics/">WebSocket</a> is a more-natural solution, designed for full duplex client–server communication&mdash;messages that can flow in both directions at the same time. One advantage of a signaling service built with pure WebSocket or server-sent events (<code>EventSource</code>) is that the backend for these APIs can be implemented on a variety of web frameworks common to most web-hosting packages for languages, such as PHP, Python, and Ruby.</p>
 
-<p>About three quarters of browsers <a href="http://caniuse.com/#search=websocket">support WebSocket</a> and, more importantly, all browsers that support WebRTC also support WebSocket, both on desktop and mobile. <a href="https://en.wikipedia.org/wiki/Transport_Layer_Security">TLS</a> should be used for all connections, to ensure messages cannot be intercepted unencrypted, and also to <a href="http://www.infoq.com/articles/Web-Sockets-Proxy-Servers">reduce problems with proxy traversal</a>. (For more information about WebSocket and proxy traversal see the <a href="http://hpbn.co/webrtc">WebRTC chapter</a> in Ilya Grigorik's <em>High Performance Browser Networking</em>. Peter Lubber's <a href="http://refcardz.dzone.com/refcardz/html5-websocket">WebSocket Cheat Sheet</a> has more information about WebSocket clients and servers.)</p>
```

Comment:
Looks like quite a lot got removed here inadvertently, from 'Peter Lubbers' to above 'It is possible ...'. 

Please reinstate :).

BTW: I just realised a mistake here: should be `Peter Lubbers'` not `Peter Lubber's`. (His name is Peter Lubbers.)

Reply:
Will fix. It should actually be Peter Lubbers's.

Reply:
It should actually be Peter Lubbers's. Will fix.